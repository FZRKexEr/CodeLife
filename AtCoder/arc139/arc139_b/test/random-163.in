100
15 16 15 2 20 10
50 1 9 18 16 11
6 12 6 15 14 13
59 4 15 2 9 16
100 1 18 13 10 5
25 10 15 12 16 5
73 3 7 11 5 2
25 15 17 1 3 9
94 11 5 7 1 17
13 14 19 6 15 20
50 10 14 10 18 8
93 14 4 16 3 15
72 1 5 3 9 10
9 9 12 17 14 15
47 1 17 8 13 7
31 13 13 16 10 12
54 14 8 11 20 14
27 19 20 3 7 9
62 7 10 12 10 12
22 11 4 11 11 7
47 2 4 7 5 1
77 14 11 8 3 6
51 11 1 12 5 1
71 16 19 10 7 9
26 16 11 9 12 9
61 8 17 7 11 15
92 12 3 17 3 19
42 9 17 2 11 7
22 19 11 6 16 18
42 11 7 3 10 18
62 20 4 7 17 4
9 13 3 19 17 10
55 16 5 17 18 2
97 10 6 8 10 19
11 9 10 1 19 14
11 13 17 20 19 4
48 19 8 16 1 16
85 18 11 16 9 16
31 8 19 19 7 17
95 12 3 10 12 13
92 10 6 12 4 3
79 18 18 20 16 8
71 7 20 12 16 11
58 6 10 9 6 5
23 2 12 11 6 20
54 12 19 2 16 11
18 17 4 20 9 4
7 1 18 7 10 9
22 17 1 2 18 8
24 11 3 3 11 2
79 12 10 19 18 1
95 4 17 5 10 1
24 15 19 17 2 3
7 13 14 17 6 9
76 5 16 9 9 7
3 7 18 8 6 8
9 5 6 14 11 7
70 2 10 19 2 13
38 10 20 1 19 5
9 14 7 2 3 14
71 1 3 2 1 11
71 11 14 5 17 11
93 19 20 5 14 5
99 8 19 14 5 13
49 9 19 2 10 14
37 16 4 10 15 15
86 14 20 15 18 16
58 16 12 4 15 6
58 12 17 12 15 4
99 6 12 4 13 6
33 5 2 16 2 10
70 16 5 1 12 4
73 16 15 20 18 13
69 13 6 20 18 16
46 4 1 2 8 20
68 13 13 18 3 5
1 2 9 17 9 9
51 17 15 11 4 20
48 7 19 16 12 18
94 14 4 20 15 14
49 4 7 5 18 16
9 14 12 2 10 16
54 17 2 13 19 13
92 3 1 9 9 19
53 4 1 1 15 5
75 12 12 19 13 18
83 4 10 9 15 6
55 10 12 14 7 12
63 15 3 4 17 18
42 18 13 6 18 6
73 20 2 7 1 14
1 18 2 8 2 8
42 5 16 18 20 15
54 2 8 6 11 6
48 3 10 13 12 15
4 1 2 17 9 11
92 7 15 9 9 15
14 10 15 16 13 15
13 3 3 9 10 16
26 5 10 8 7 20
