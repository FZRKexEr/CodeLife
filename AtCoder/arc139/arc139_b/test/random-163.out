10
135
13
76
125
21
25
13
32
78
72
64
146
14
118
100
90
30
84
22
23
42
27
42
54
70
93
30
34
54
65
30
22
117
11
220
50
84
100
143
38
172
97
38
61
46
29
49
28
16
94
35
88
119
64
24
49
70
23
18
71
60
85
63
42
55
138
50
38
60
38
53
252
134
92
69
17
12
82
158
112
18
83
288
53
135
75
59
80
36
94
8
70
69
110
22
108
77
49
43
